test/standard/runners/lsp_test.rb

```diff
@@ -66,21 +65,6 @@ def test_did_open
     }, msgs.first)
   end
 
-  def test_diagnotic_route
-    msgs, err = run_server_on_requests({
-      method: "textDocument/diagnostic",
-      jsonrpc: "2.0",
-      params: {
-        textDocument: {
-          uri: "file:///path/to/file.rb"
-        }
-      }
-    })
-
-    assert_equal "", err.string
-    assert_equal 0, msgs.count
-  end
```

Comment:
Is there any intentions to remove rather than update the test case as follows?

```diff
  def test_diagnotic_route
    msgs, err = run_server_on_requests({
      method: "textDocument/diagnostic",
      jsonrpc: "2.0",
      params: {
        textDocument: {
          uri: "file:///path/to/file.rb"
        }
      }
    })

-   assert_equal ", err.string
-   assert_equal 0, msgs.count
+   assert_equal "[server] Unsupported Method: textDocument/diagnostic\n", err.string
+   assert_equal 1, msgs.count
  end
```

Reply:
I'm not sure that spec makes sense like that. Since we don't advertise support for it, a client won't ever send us that message.